yrs/src/block.rs

```diff
@@ -1039,10 +1042,7 @@ impl ItemContent {
     pub fn len(&self) -> u32 {
         match self {
             ItemContent::Deleted(deleted) => *deleted,
-            ItemContent::String(str) => {
-                // @todo this should return the length in utf16!
-                str.len() as u32
-            }
+            ItemContent::String(str) => str.chars().count() as u32,
```

Comment:
For sake of proper universal unicode support, we could a string length by the number of characters rather than byte length of a string.